src/main/webapp/web.xml

```diff
@@ -3,13 +3,125 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
          version="3.0">
+
     <servlet>
-        <servlet-name>index</servlet-name>
+        <servlet-name>taxi-service</servlet-name>
         <servlet-class>core.controller.IndexController</servlet-class>
     </servlet>
     
     <servlet-mapping>
-        <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
+        <servlet-name>taxi-service</servlet-name>
+        <url-pattern>/</url-pattern>
     </servlet-mapping>
+
+    <servlet>
+        <servlet-name>addDriver</servlet-name>
+        <servlet-class>core.controller.driver.AddDriverController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>addDriver</servlet-name>
+        <url-pattern>/driver/add</url-pattern>
```

Comment:
```suggestion
        <url-pattern>/drivers/add</url-pattern>
```
it is recommended to use plural names in URLs

Reply:
A bit of refactoring and we are there